### PR TITLE
Switching Assistant: show external insurer display name

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

See title.

## Justify why they are needed

The API doesn't guarantee that we have an external insurer but it shouldn't happen. That's why I added the datadog logging.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1434** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1434" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1434?utm_source=stack-comment).

File: apps/store/src/components/SwitchingAssistantPage/SwitchingAssistantPage.tsx [hunk 1]
```diff
@@ -64,9 +71,7 @@ export const SwitchingAssistantPage = ({ shopSession }: Props) => {
                     </Pill>
                     <Space y={1}>
                       <div>
-                        <Text>
-                          {item.name} · {item.company}
-                        </Text>
+                        <Text>{[item.name, item.company].filter(Boolean).join(' · ')}</Text>
```
Comment (alebedev) [inline apps/store/src/components/SwitchingAssistantPage/SwitchingAssistantPage.tsx:74]: Do we want to extract util function?  There're multiple places we're showing existing insurance

**Review by alebedev — APPROVED** [review]
Looks good

File: apps/store/src/components/SwitchingAssistantPage/SwitchingAssistantPage.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/components/SwitchingAssistantPage/SwitchingAssistantPage.tsx:74]: hmm not sure what you mean -- what would the util function do?

**Comment by robinandeer** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: nAqvJu8ea68oKZxSWyRw.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1434)

**Pull request merged**